lib/registrant.rb

```diff
@@ -0,0 +1,23 @@
+class Registrant 
+    attr_reader :name, :age, :permit, :license_data
```

Comment:
Careful here, we don't want to create a method called `permit` since we have the method `permit?` that does the same function and follows Ruby convention.